src/Sylius/Bundle/ApiBundle/spec/Doctrine/QueryItemExtension/OrderMethodsItemExtensionSpec.php

```diff
@@ -302,6 +302,43 @@ function it_applies_conditions_to_shop_select_payment_method_operation_with_give
         );
     }
 
+    function it_applies_conditions_to_shop_account_change_payment_method_operation_with_given_user(
+        UserContextInterface $userContext,
+        QueryBuilder $queryBuilder,
+        ShopUserInterface $shopUser,
+        CustomerInterface $customer,
+        QueryNameGeneratorInterface $queryNameGenerator
+    ): void {
+        $queryBuilder->getRootAliases()->willReturn(['o']);
+
+        $userContext->getUser()->willReturn($shopUser);
+
+        $shopUser->getCustomer()->willReturn($customer);
+        $customer->getId()->willReturn(1);
+        $shopUser->getRoles()->willReturn(['ROLE_USER']);
+
+        $queryBuilder
+            ->andWhere(sprintf('%s.customer = :customer', 'o'))
+            ->shouldBeCalled()
+            ->willReturn($queryBuilder)
+        ;
+
+        $queryBuilder
+            ->setParameter('customer', 1)
+            ->shouldBeCalled()
+            ->willReturn($queryBuilder)
+        ;
+
+        $this->applyToItem(
+            $queryBuilder,
+            $queryNameGenerator,
+            OrderInterface::class,
+            ['tokenValue' => 'xaza-tt_fee'],
+            'shop_account_change_payment_method',
+            [ContextKeys::HTTP_REQUEST_METHOD_TYPE => Request::METHOD_PATCH],
+            );
```

Comment:
```suggestion
        );
```
We need to run some CS fixer in the nearest days